.gitignore

```diff
@@ -7,9 +7,6 @@
 # Ignore bundler config.
 /.bundle
 
-# Ignore the default SQLite database.
```

Comment:
Don't know what the `mysql` equivalents are that need to be ignored. We can `.gitignore` them as we see them.

Reply:
SQLite creates a separate db per project, in the project directory, but MySQL is on its own server, so the MySQL database shouldn't show up in the repo.